Sources/UIKitExtensions/UIToolBar+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIToolBarFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+import Foundation
+import UIKit
+public extension ConfigurationSet where Base: UIToolbar {
+    
+    func setItems(_ items: [UIBarButtonItem]?, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UIToolbar: Base ) in
```

Comment:
This function is only useful for existing UIToolbar whose items change should be animated.
The items Property ist writable directly so there is no need for this function.